app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/app/tracking/RealAppTrackingProtection.kt

```diff
@@ -60,4 +62,10 @@ class RealAppTrackingProtection @Inject constructor(
             vpnFeaturesRegistry.unregisterFeature(AppTpVpnFeature.APPTP_VPN)
         }
     }
+
+    override suspend fun isAppExcluded(packageName: String): Boolean {
+        return withContext(dispatcherProvider.io()) {
```

Comment:
nit: should we already define the dispatcher here? we know is disk IO

Reply:
@aitorvs Didn’t I already do that?